src/adapters/peggedAssets/usd-coin/index.ts

```diff
@@ -342,6 +343,10 @@ const chainContracts: ChainContracts = {
   },
   base: {
     bridgedFromETH: ["0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA"], //
+  },
+  kujira: {
+    bridgedFromAxelar: ["ibc/295548A78785A1007F232DE286149A6FF512F180AF5657780FC89C009E2C348F"],
```

Comment:
Could you try bridgedFromETH instead of bridgedFromAxelar
Same with line 349

Reply:
Sure thing. What should L349 be? Happy to have them both as fromETH, or do you want the distinction for Noble USDC in the Cosmos ecosystem?